sources/packages/web/src/main.ts

```diff
@@ -29,9 +29,20 @@ import Dropdown from "primevue/dropdown";
 import InputNumber from "primevue/inputnumber";
 import ProgressSpinner from "primevue/progressspinner";
 import Chip from "primevue/chip";
+import TabMenu from "primevue/tabmenu";
 import { AppConfigService } from "./services/AppConfigService";
 import configValidationRules from "./validators/ValidatorConfigUI";
 
+import { library } from "@fortawesome/fontawesome-svg-core";
+import {
+  faMapPin,
+  faCheck,
+  faCircle,
+  faSearch,
+} from "@fortawesome/free-solid-svg-icons";
```

Comment:
Are these icons not present in vue-fontawsome and that is why we importing it from font-awesome?

Reply:
these are ` vue font-awesome`  icons, we can import `all icons` or we can import only required icons. I was going for the 2nd method. let me know @dev thoughts

Reply:
FYI, the dev @ here is pointing to some other group I guess. 😊
Yes we can talk about it, when we sync up.

Reply:
ref - https://www.npmjs.com/package/@fortawesome/vue-fontawesome 
![image](https://user-images.githubusercontent.com/77353155/145640265-27615f29-571f-4519-8ddd-f0f38c557656.png)